cmd/create_access_token_service/main.py

```diff
@@ -0,0 +1,100 @@
+from botocore.exceptions import ClientError
+
+from inquirer import List, Password, Text, prompt
+from termcolor import colored
+
+logging.basicConfig()
+LOGGER = logging.getLogger("create-access-token-service")
+LOGGER.setLevel(logging.INFO)
+
+CLIENT_CREDENTIALS_CLIENT_ID_SSM_KEY = (
+    "/paddle/%s/cognito_sellers/internal_service_client_id"
+)
+CLIENT_CREDENTIALS_CLIENT_SECRET_SSM_KEY = (
+    "/paddle/%s/cognito_sellers/internal_service_client_secret"
+)
+
+
+def prompt_choices() -> list:
+    questions = [List("environment", message="Environment:", choices=["local"])]
+    answers = prompt(questions, raise_keyboard_interrupt=True)
+
+    return answers
+
+
+def find_parameter(parameters: list, name: str, environment: str) -> str:
+    for (index, item) in enumerate(parameters):
+        if item["Name"] == name % environment:
+            return item.get("Value")
+
+    return None
+
+
+def fetch_ssm_parameters(session: boto3_session, environment: str) -> dict:
+    ssm_client = session.client("ssm")
+    ssm_keys = [
+        CLIENT_CREDENTIALS_CLIENT_ID_SSM_KEY % environment,
+        CLIENT_CREDENTIALS_CLIENT_SECRET_SSM_KEY % environment,
+    ]
+
+    response = ssm_client.get_parameters(Names=ssm_keys, WithDecryption=True)
+    client_id = find_parameter(
+        response.get("Parameters", {}),
+        CLIENT_CREDENTIALS_CLIENT_ID_SSM_KEY,
+        environment,
+    )
+    client_secret = find_parameter(
+        response.get("Parameters", {}),
+        CLIENT_CREDENTIALS_CLIENT_SECRET_SSM_KEY,
+        environment,
+    )
+
+    return {"client_id": client_id, "client_secret": client_secret}
+
+
+def generateAccessToken(environment: str) -> dict:
+    session = boto3_session(
+        profile_name="development" if environment == "local" else environment
+    )
+
+    client_parameters = fetch_ssm_parameters(session, environment)
+
+    # Build the auth header
+    authHeader = (
+        client_parameters.get("client_id")
+        + ":"
+        + client_parameters.get("client_secret")
+    )
+    encodedAuthHeader = base64.b64encode(authHeader.encode("utf-8"))
+
+    # Prepare the request
+    # TODO:make it support more than local environment
+    url = "https://paddle-local-sellers.auth.us-east-1.amazoncognito.com/oauth2/token"
+    payload = "grant_type=client_credentials&scope=paddle.local%2Fall%3Ainternal"
+    headers = {
+        "Content-Type": "application/x-www-form-urlencoded",
+        "Authorization": "Basic " + encodedAuthHeader.decode("utf-8"),
+        "cache-control": "no-cache",
+    }
+    response = requests.request("POST", url, data=payload, headers=headers)
+    json_data = json.loads(response.text)
+    return json_data.get("access_token")
+
+
+def handle() -> None:
+    try:
+        answers = prompt_choices()
+        token = generateAccessToken(answers["environment"])
+        LOGGER.info("Successfully created access token: \n%s", token)
+    except Exception as exception:
+        LOGGER.error(exception)
```

Comment:
You might want to cast this exception to string to get a readable error
```suggestion
        LOGGER.error(str(exception))
```

Reply:
thanks, done